nrf_wifi/fw_if/umac_if/src/default/fmac_api.c

```diff
@@ -60,7 +60,7 @@ static enum nrf_wifi_status nrf_wifi_fmac_init_tx(struct nrf_wifi_fmac_dev_ctx *
 		def_priv->data_config.max_tx_aggregation *
 		sizeof(struct nrf_wifi_fmac_buf_map_info));
 
-	def_dev_ctx->tx_buf_info = nrf_wifi_osal_mem_zalloc(fmac_dev_ctx->fpriv->opriv,
+	def_dev_ctx->tx_buf_info = nrf_wifi_osal_pkt_mem_zalloc(fmac_dev_ctx->fpriv->opriv,
```

Comment:
My preference would be to use `data` instead of `pkt`, but not a requirement.

Reply:
Also shouldn't there be a corresponding `ctrl` variant now?